### DISCUSSION TITLE
Is there anyway to show multiple subplots that aren't rendered together?

### DISCUSSION THREAD

**Discussion by DYT-Flyer** [marcomusy/vedo#456] [Q&A] [answered]
I want to be able to show two different objects and interact with one and not the other.

**Comment by DYT-Flyer** [comment, marked as answer]
Found this: https://github.com/marcomusy/vedo/blob/master/examples/advanced/multi_viewer1.py

Works now.

**Reply by marcomusy** [reply]
if it's a matter of moving around individual objects you also have the option:
`show(my_objs, mode=1)`
